src/startpage/startpage.cpp

```diff
@@ -101,7 +87,7 @@ StartPage::StartPage(QWidget *parent) :
     setLayout(pWdwLayout);
 
     // resizing & centring dialog
-    setGeometry(geometry().x(), geometry().y(), 500, 300);
+    setGeometry(geometry().x(), geometry().y(), 500, 250);
```

Comment:
Magic numbers is bad practice.

Reply:
fixed that
Now widgets geometry is set according to screen geometry